styles/style.css

```
@@ -0,0 +1,330 @@
+}
+
+header > div > p {
+    text-align: center;
+    font-size: 19px;
+    color: whitesmoke;
+}
+
+header > div > button {
+    background-color: #ED4385;
+    color: whitesmoke;
+    padding: 15px 30px;
+    border-radius: 40px;
+    border: 0;
+    box-shadow: 1px 2px 5px #3a3a3a;
+}
+
+header > img {
+    margin-top: 80px;
+}
+
+/* --------------- 4.2 Introduction ------------------- */
+
+.hello > div {
+    max-width: 600px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    gap: 20px;
+    margin: 80px 40px;
+}
+
+.hello h3 {
+    text-align: center;
+    color: #5722CD;
+}
+
+.hello p {
+    text-align: center;
+    margin-top: 60px;
+}
+
+.hello a {
+    color: #5722CD;
+}
+
+/* --------------- 4.3 Work ------------------- */
+
+.work {
+    background-color: #F3F6F9;
+}
+
+.work > div {
+    display: flex;
+    flex-direction: column;
+}
+
+.work > div > article {
+    display: flex;
+    flex-direction: row;
+    gap: 60px;
+    margin-top: 60px;
+    padding: 40px;
+}
+
+.work > div > article:nth-child(2) {
+    display: flex;
+    flex-direction: row-reverse;
+}
+
+.work > div > article:nth-child(3) {
+    margin-bottom: 80px;
+}
+
+.work > div > article > div {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 30px;
+    width: 300px;
+}
+
+.work > div > article > div > h2 {
+    font-size: 32px;
+}
+
+.work > div > article > div > img {
+    width: 60px;
+}
+
+.work > div > article > div > span {
+    flex-basis: 800px;
+    flex-grow: 0;
+    flex-shrink: 1;
+}
+
+.img-wrapper > img {
+    width: 100%;
+    box-shadow: 0 20px 30px #9b9b9b;
+}
+
+/* --------------- 4.3 Services ------------------- */
+
+.service > article {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    max-width: 1500px;
+    padding: 80px 40px;
+    flex-wrap: wrap;
+    gap: 40px
+}
+
+.service > article > div {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    flex-basis: 350px;
+    padding: 40px;
+    border: 1px solid #5722CD;
+    box-shadow: 0 20px 30px #9b9b9b;
+}
+
+.service img {
+    width: 80px;
+}
+
+.service h3 {
+    color: #5722CD;
+}
+
+/* --------------- 4.3 Form ------------------- */
+
+.form {
+    background-image: linear-gradient(120deg, #4845E4, #5722CD);
+    color: whitesmoke;
+    display: flex;
+    flex-direction: column;
+}
+
+.form > div {
+    max-width: 800px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 20px;
+    margin: 60px 40px;
+}
+
+.form > div > h1 {
+    font-size: 40px;
+}
+
+
+.form > div > form {
+    display: flex;
+    flex-direction: row;
+}
+
+.form > div > form > fieldset {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 10px;
+    border-style: none;
+}
+
+.form > div > form > fieldset > input {
+    flex-basis: 40px;
+    border-radius: 2px;
+    width: 500px;
```

Als de flex-direction row is (dat is het by default), dan zul je geen flex-basis in combinatie met width gebruiken. Gebruik óf flex-basis óf width.
```suggestion
    flex-basis: 500px;
    flex-grow: 0;
    flex-shrink: 0;
```
Of als je het nog korter wilt schrijven
```suggestion
    flex: 0 0 500px;
```